libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/SignupGoogleFragment.java

```diff
@@ -35,10 +33,9 @@
 
     public static final String TAG = "signup_google_fragment_tag";
 
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        ((WordPress) getActivity().getApplication()).component().inject(this);
+    @Override public void onAttach(Context context) {
+        super.onAttach(context);
```

Comment:
`AndroidSupportInjection.inject(this);` needs to come above `super.onAttach()` (per the [Dagger docs](https://google.github.io/dagger/android.html)).

Reply:
I moved the statement in 883bafb.